querydsl-sql/src/main/java/com/querydsl/sql/SQLTemplates.java

```diff
@@ -250,14 +252,15 @@ public SQLTemplates build() {
 
     @Deprecated
     protected SQLTemplates(String quoteStr, char escape, boolean useQuotes) {
-        this(Keywords.DEFAULT, quoteStr, escape, useQuotes);
+        this(Keywords.DEFAULT, quoteStr, escape, useQuotes, false);
     }
 
-    protected SQLTemplates(Set<String> reservedKeywords, String quoteStr, char escape, boolean useQuotes) {
+    protected SQLTemplates(Set<String> reservedKeywords, String quoteStr, char escape, boolean useQuotes, boolean requiresSchemaInWhere) {
```

Comment:
Add in an additional constructor matching the old one.
For backwards compatibility.